dev/php/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM php:8.1-fpm-alpine
```

```suggestion
FROM php:8.2-fpm-alpine
```
Используем актуальные версии

---

а в тз 8.1 же